File: docs/assets/images/textures/indexer.py
# Writes the names of all textures in this directory to the 'index.json' file, to be used in the app later.
import os, json
index = open('index.json', "w")
files = os.listdir()
images = [ f for f in files if f.endswith('.jpg') ]
index.write("{\"images\":" + json.dumps(images) + "}")
index.close()

File: src/assets/images/tech/indexer.js
// Writes the names of all logos in this directory to the 'index.json' file, to be used in the app later.
const fs = require('fs');

fs.readdir(__dirname, (_, files) => {
    files = files.filter(x => x.endsWith('.svg'));
    output = {icons: files}
    fs.writeFileSync('index.json', JSON.stringify(output), 'utf-8') 
});